code/game/objects/structures/mirror.dm

```diff
@@ -91,6 +94,22 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/mirror/broken, 28)
 
 	return display_radial_menu(user)
 
+/obj/structure/mirror/wrench_act_secondary(mob/living/user, obj/item/tool)
+	. = ..()
```

Comment:
Parent does nothing

Reply:
I was thinking the same, was referencing another structure with this proc and it felt off.
Coded it at like 2am though so I wasn't going to risk slipping up